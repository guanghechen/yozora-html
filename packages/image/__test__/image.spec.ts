import type { Image } from '@yozora/ast'
import { ImageType } from '@yozora/ast'
import renderImage from '../src'

describe('snapshot', function () {
  it('basic', function () {
    const node: Image = {
      type: ImageType,
      url: 'https://github.com/favicon.svg/',
      alt: 'github favicon',
      title: 'github favicon',
    }
    expect(renderImage(node)).toMatchSnapshot()
  })
})
