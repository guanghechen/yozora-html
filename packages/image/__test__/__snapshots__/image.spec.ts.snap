// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot basic 1`] = `"<figure class=\\"yozora-image\\"><img alt=\\"github favicon\\" src=\\"https://github.com/favicon.svg/\\" title=\\"github favicon\\" /><figcaption>github favicon</figcaption></figure>"`;
